### ISSUE TITLE
Add the Yes to all - button to delete all conflicted files

### ISSUE THREAD

**Issue by popos123** [nextcloud/desktop#7446]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

## Feature request

**Which Nextcloud Version are you currently using:** 29

**Is your feature request related to a problem? Please describe.**

I'm frustrated witch clicking 230 times to Yes button

![Image](https://github.com/user-attachments/assets/f370cebb-25d4-4ca0-b766-36aeea36a7f7)

**Describe the solution you'd like**

Just create YES to all files - a checkbox or button.

**Comment by ted423** [comment]
I found I can search conflicted copy and delete them all without click in nextcloud

**Comment by popos123** [comment]
Yes, it is a good workaround. But the problem / bug is still existing.
> I found I can search conflicted copy and delete them all without click in nextcloud

**Comment by ted423** [comment]
yeah, my problem is that there are always thousands of files to handle each time. If I can't find a solution to this, I'll have no choice but to leave Nextcloud. Currently, it seems the developers don't have sufficient capacity to address the numerous issues
